src/lib/simulator/physics/Gravity.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "Particle.h"
+namespace simulator::physics {
+
+/**
+ * Implements physics for simulations using gravity as force Model
+ *
+ * */
+class Gravity {
+ public:
+  Gravity() = delete;
+  auto static calculate_force(const Particle &particle1, const Particle &particle2) -> std::array<double, 3>;
```

Comment:
why static? and why not just use `std::array<double,3>` instead of `auto` and remove `-> std::array<double, 3>`?